src/Seq.Input.HealthCheck/HttpHealthCheck.cs

```diff
@@ -72,6 +92,25 @@ public async Task<HealthCheckResult> CheckNow(CancellationToken cancel)
             {
                 var request = new HttpRequestMessage(HttpMethod.Get, probedUrl);
                 request.Headers.Add("X-Correlation-ID", probeId);
+                if(_optionalHeader != null) {
```

Comment:
Perhaps reusing the code from _Seq.App.HttpRequest_ would help keep complexity down in this class? E.g. https://github.com/datalust/seq-app-httprequest/blob/e23b2328141352f5b4851c7502801f5bfe6511f7/src/Seq.App.HttpRequest/Settings/HeaderSettingFormat.cs#L9 provides request header parsing that handles the two settings accepted by the app.

Reply:
That is good, however do you suggest copying the code?
I have no problems with that.